resources/prometheus/prometheus-rules.yaml

```diff
@@ -11,42 +11,42 @@ spec:
   - name: k8s.rules
     rules:
     - expr: |
-        sum(rate(container_cpu_usage_seconds_total{job="kubelet", image!="", container_name!=""}[5m])) by (namespace)
+        sum(rate(container_cpu_usage_seconds_total{job="kubelet", image!="", container!="POD"}[5m])) by (namespace)
```

Comment:
Also I cannot say I understand these changes:
  * did prometheus  field names change here (`container_name` -> `container`)?
  * what does the "POD" value mean and why it was empty before?

Reply:
In kubelet 1.17+, I assume they changed the label name for container. POD values mean metric for whole pod, not for one of the containers. But maybe I am wrong, didn't compare sum and POD metric, to be honest. All that calculations for the same cpu usage are different in different graphs and me, sometimes, have no idea how it works. I just want to use this one, because I understand it. Well, mostly.